### PR TITLE
Update README URLs based on HTTP redirects

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein
### HTTPS Corrected URLs

| Was | Now |
| --- | --- |
| http://twitter.com/jenkinsci | https://twitter.com/jenkinsci |
### Other Corrected URLs

| Was | Now |
| --- | --- |
| http://jenkins-ci.org/sites/default/files/jenkins_logo.png | https://jenkins.io/sites/default/files/jenkins_logo.png |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,12 +53,12 @@ Jenkins is **licensed** under the **[MIT License]**. The terms of the license ar
     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
     THE SOFTWARE.
 
-[ButlerImage]: http://jenkins-ci.org/sites/default/files/jenkins_logo.png
+[ButlerImage]: https://jenkins.io/sites/default/files/jenkins_logo.png
 [MIT License]: https://github.com/jenkinsci/jenkins/raw/master/LICENSE.txt
 [Mirrors]: http://mirrors.jenkins-ci.org
 [GitHub]: https://github.com/jenkinsci/jenkins
 [website]: http://jenkins-ci.org
```
Comment (oleg-nenashev) [inline README.md]: In such case this guy should be changed as well

Comment (ReadmeCritic) [inline README.md]: @oleg-nenashev what change would you like, please clarify. thx

Comment (oleg-nenashev) [inline README.md]: The new website is https://jenkins.io/ . All http://jenkins-ci.org are outdated. It seems to be aligned with your HTTP => HTTPS fixes

**Comment by ReadmeCritic** [comment]
@oleg-nenashev will close this and reopen ! :sweat_smile:

**Pull request closed without merging**